packages/gui/src/components/offers2/CreateOfferBuilder.tsx

```diff
@@ -17,17 +17,20 @@ type createDefaultValuesParams = {
   walletType?: WalletType; // CAT or STANDARD_WALLET (XCH), indicates whether a token or CAT has a default entry
   assetId?: string; // Asset ID of the CAT
   nftId?: string; // NFT to include in the offer by default
+  nftIds?: string[]; // multiple NFT selection
   nftWalletId?: number; // If set, indicates that we are offering the NFT, otherwise we are requesting it
```

Comment:
This is sort of a mess. Since we're now dealing with (possibly) multiple NFTs coming from different NFT wallets, we need to have a pair of NFT ID along with its wallet ID.

I'd recommend replacing these 3 props with a single `nftIdsAndWalletIds` prop that contains an array of `{ nftId: string, walletId: number }` objects.

Let's revisit this after merging this PR since it will be a bigger change.

Reply:
Tracked as https://github.com/Chia-Network/chia-blockchain-gui/issues/1252